exchanges/gateio/gateio.go

```diff
@@ -3686,3 +3709,15 @@ func getSettlementFromCurrency(currencyPair currency.Pair) (settlement currency.
 		return currency.EMPTYCODE, fmt.Errorf("%w %v", errCannotParseSettlementCurrency, currencyPair)
 	}
 }
+
+// GetAccountDetail retrieves account details
+func (g *Gateio) GetAccountDetail(ctx context.Context) (*AccountDetail, error) {
+	var resp *AccountDetail
```

Comment:
* 🐒  I'd prefer to patch over their API spelling mistakes at our end. If someone searches for accountDetail they'll still find this.
* 🦍 (_This nitpick is bigger, but still optional_) I think the type should definitely be fixed
```suggestion
func (g *Gateio) GetAccountDetails(ctx context.Context) (*AccountDetails, error) {
```